### PR TITLE
Fix for the pot_plugpackage_path variable

### LINKED ISSUE #15275: Missing separator
https://github.com/vim/vim/blob/22105fd1fe0dcfe993b5c04c6ebe017a626116e3/src/po/Makefile#L320

### PR DESCRIPTION
**Pull request by RestorerZ**
Fix for the pot_plugpackage_path variable
Fix #15275

### CODE REVIEW

File: src/po/Makefile [hunk 1]
```diff
@@ -313,8 +313,9 @@ $(LANGUAGES):
 # For translations of plug-ins
 #######
 
+POT_PLUGPACKAGE_PATH = ${PWD}
```
Comment (k-takata) [inline src/po/Makefile]: Any reason to use `${}` instead of `$()`? (Both are the same, though.)

File: src/po/Makefile [hunk 2]
```diff
@@ -327,7 +328,6 @@ $(PLUGPACKAGE).pot: $(PO_PLUG_INPUTLIST)
 	rm -f *.js ./vim_to_js
 
 # Converting the PO file of the plug-in package to the binary format of the MO
-MO_PLUGPACKAGE_PATH != pwd
 $(PLUGPACKAGE).mo: $(PO_PLUGPACKAGE)
 	$(MSGFMTCMD) -o $(MO_PLUGPACKAGE_PATH)/$@ $<
```
Comment (k-takata) [inline src/po/Makefile:332]: Should be `POT_PLUGPACKAGE_PATH` instead of `MO_PLUGPACKAGE_PATH`?

**Comment by chrisbra** [comment]
Wondering if this would be simpler to get rid of the `...PLUGPACKAGE_PATH` variable completly (I assume xgettext and msgfmt will use the current directory by default anyhow?):
```diff
diff --git a/src/po/Makefile b/src/po/Makefile
index 0f7f85dd1..caec2fe62 100644
--- a/src/po/Makefile
+++ b/src/po/Makefile
@@ -314,21 +314,18 @@ $(LANGUAGES):
 #######

 # Preparing the POT file of the plug-in package
-POT_PLUGPACKAGE_PATH != pwd
 $(PLUGPACKAGE).pot: $(PO_PLUG_INPUTLIST)
        $(VIMPROG) -u NONE --not-a-term -S tojavascript.vim \
                $(PLUGPACKAGE).pot $?
        $(XGETTEXT) --from-code=UTF-8 --default-domain=$(PLUGPACKAGE) \
                --package-name=$(PLUGPACKAGE) \
-               --output-dir=$(POT_PLUGPACKAGE_PATH) \
                --output=$(PLUGPACKAGE).pot --files-from=./vim_to_js
        $(VIMPROG) -u NONE --not-a-term -S fixfilenames.vim \
-               $(POT_PLUGPACKAGE_PATH)/$(PLUGPACKAGE).pot $?
+               $(PLUGPACKAGE).pot $?
        rm -f *.js ./vim_to_js

 # Converting the PO file of the plug-in package to the binary format of the MO
-MO_PLUGPACKAGE_PATH != pwd
 $(PLUGPACKAGE).mo: $(PO_PLUGPACKAGE)
-       $(MSGFMTCMD) -o $(MO_PLUGPACKAGE_PATH)/$@ $<
+       $(MSGFMTCMD) -o $@ $<

 # vim: set noet sw=8 ts=8 sts=0 wm=0 tw=0 ft=make:
```

File: src/po/Makefile (hunk 1 above)
Comment (RestorerZ) [reply src/po/Makefile]: As I understand it, the *BSD world prefers curly braces. And the GNU world also, in principle, uses them for variables.
But in this makefile, parentheses are used everywhere, so I fixed it. A uniform style must be observed.

File: src/po/Makefile (hunk 2 above)
Comment (RestorerZ) [reply src/po/Makefile:332]: Yes, I hurried, inattentive.

**Comment by RestorerZ** [comment]
> Wondering if this would be simpler to get rid of the `...PLUGPACKAGE_PATH` variable completly (I assume xgettext and msgfmt will use the current directory by default anyhow?):

Probably, yes. I haven't tested it, but I think you're right.
I'll do it now.

**Comment by RestorerZ** [comment]
Although not. We have declared the variable "MO_PLUGPACKAGE_PATH" and "POT_PLUGPACKAGE_PATH" in the documentation. This is done in case you need to place the result not in the "src/po" directory (which is for translations of the Vim editor itself), but where it is needed by the one who translates the scripts.
If you don't mind, I'll just fix what Takata‐san pointed out and send the push here.

**Comment by chrisbra** [comment]
sure

**Comment by chrisbra** [comment]
thanks!

**Pull request closed without merging**